bytearray_operations_test.go

```diff
@@ -72,6 +72,161 @@ func TestLeftShift(t *testing.T) {
 	}
 }
 
+var testcasesMask = []struct {
+	name   string
+	data   []byte
+	mask   []byte
+	result []byte
+}{
+  {"data and mask of equal length", []byte{0xDA, 0x99, 0xBA}, []byte{0xAD, 0x11, 0xAB}, []byte{0x88, 0x11, 0xAA}},
```

Comment:
Don't forget to format the code with `go fmt ./...` (by example)

Reply:
Sorry couldn't tell apart tabs and spaces on my editor. Will fix that.